src/board.cpp

```
@@ -0,0 +1,45 @@
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with Zagreus.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "board.h"
+
+namespace Zagreus {
+bool Board::isMoveLegal(Move move) const {
+    // TODO: Implement
+}
```

cppcheck

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)